api/src/main/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadService.java

```diff
@@ -14,6 +15,8 @@
 
     List<Vehicle> findByRegistration(String registration);
 
+    List<DvlaVehicle> findDvlaVehicleByRegistration(String registration);
```

Comment:
Why would this add multiple additional models? You would just need one JSON model, DvlaVehicle no?